tests/baselines/reference/tsxElementResolution3.errors.txt

```diff
@@ -17,4 +17,4 @@ tests/cases/conformance/jsx/file.tsx(12,2): error TS2322: Type '{ w: string; }'
     <span w='err' />;
      ~~~~
 !!! error TS2322: Type '{ w: string; }' is not assignable to type '{ n: string; }'.
-!!! error TS2322:   Property 'n' is missing in type '{ w: string; }'.
\ No newline at end of file
+!!! error TS2322:   Property 'w' does not exist on type '{ n: string; }'.
```

Comment:
The directionality of this error changed, meaning that maybe source swapped with target. Is that correct?

Reply:
Ehhhhh, not quite. Previous JSX didn't use normal excess-property-checking, and so the normal assignability error would be reported. Excess property errors in the usual codepath are reported at a higher priority than typical assignability errors, while in the old JSX code, excess properties only got checked if the type was otherwise assignable.